### PR TITLE
Individual school hddf 1485

### PR DESCRIPTION
**Pull request by pitabash-eGov**

### CODE REVIEW

File: apps/health_campaign_field_worker_app/lib/pages/beneficiary/deliver_intervention.dart
Comment (naveenr-egov) [inline apps/health_campaign_field_worker_app/lib/pages/beneficiary/deliver_intervention.dart:1]: dart format this file

File: apps/health_campaign_field_worker_app/lib/widgets/member_card/member_card.dart [hunk 1]
```diff
@@ -246,38 +251,57 @@ class MemberCard extends StatelessWidget {
...
-                                final bloc =
-                                    context.read<HouseholdOverviewBloc>();
+                              onPressed:
+                                  //TODO: develop
+                                  isAdverseEffect
+                                      ? () async {
+                                          await context.router.push(
+                                            SideEffectsRoute(
+                                              tasks: [
+                                                tasks!.last,
```
Comment (naveenr-egov) [inline apps/health_campaign_field_worker_app/lib/widgets/member_card/member_card.dart:261]: change it to lastOrNull

File: apps/health_campaign_field_worker_app/lib/widgets/member_card/member_card.dart [hunk 2]
```diff
@@ -246,38 +251,57 @@ class MemberCard extends StatelessWidget {
                               //   left: kPadding / 2,
                               //   right: kPadding / 2,
                               // ),
-                              onPressed: () {
-                                final bloc =
-                                    context.read<HouseholdOverviewBloc>();
+                              onPressed:
+                                  //TODO: develop
```
Comment (naveenr-egov) [inline apps/health_campaign_field_worker_app/lib/widgets/member_card/member_card.dart:255]: Remove unrequired todo

**Review by naveenr-egov — CHANGES_REQUESTED** [review]
Resolve comments

**Review by naveenr-egov — APPROVED** [review]

File: apps/health_campaign_field_worker_app/lib/widgets/member_card/member_card.dart (hunk 1 above)
Comment (pitabash-eGov) [reply apps/health_campaign_field_worker_app/lib/widgets/member_card/member_card.dart]: resolved.

File: apps/health_campaign_field_worker_app/lib/widgets/member_card/member_card.dart (hunk 2 above)
Comment (pitabash-eGov) [reply apps/health_campaign_field_worker_app/lib/widgets/member_card/member_card.dart]: resolved

File: apps/health_campaign_field_worker_app/lib/pages/beneficiary/deliver_intervention.dart
Comment (pitabash-eGov) [reply apps/health_campaign_field_worker_app/lib/pages/beneficiary/deliver_intervention.dart:1]: resolved

**Pull request merged**